avm/res/fabric/capacity/main.bicep

```diff
@@ -0,0 +1,110 @@
+metadata name = 'Fabric Capacities'
+metadata description = '<Add description>'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. Name of the resource to create.')
+param name string
+
+@description('Optional. Location for all Resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Tags of the resource.')
+param tags object?
+
+@allowed([
+  'F2'
+  'F4'
+  'F8'
+  'F16'
+  'F32'
+  'F64'
+  'F128'
+  'F256'
+  'F512'
+  'F1024'
+  'F2048'
+])
+@description('Optional. SKU tier of the Fabric resource.')
+param skuName string = 'F2'
+
+@description('Optional. SKU name of the Fabric resource.')
+param skuTier string = 'Fabric'
```

Comment:
Should this also have allowed values?

Reply:
Updated. There's only 1 SKU that I know of for now.